integration_tests/seeds/sf_task_history_data.csv

```diff
@@ -0,0 +1,11 @@
+_fivetran_active,_fivetran_start,_fivetran_end,who_id,call_disposition,owner_id,subject,description,last_modified_date,what_count,_fivetran_synced,call_object,is_deleted,last_modified_by_id,system_modstamp,id,call_type,is_high_priority,is_closed,activity_date,priority,who_count,account_id,call_duration_in_seconds,created_by_id,created_date,what_id,task_subtype,status,type,record_type_id,completed_date_time,is_archived
```

Comment:
Let's remove this since we are not using it in the end models anymore.

Reply:
Removed